testprojects/3rdparty/python/BUILD

```diff
@@ -1,4 +1,21 @@
 # Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-python_requirements()
+# TODO(#14268): Once python_requirements passes down `tags` and `compatible_resolves`, we can DRY
+#  this again with that target generator.
+common_kwargs = dict(
+    experimental_compatible_resolves=["python-testprojects"],
+    tags=["lockfile_ignore"],
```

Comment:
Ok, so its just this bit that fixes the bug / feature / change I saw.

Reply:
Correct. And then the line above is how we will fix things once we switch to using `resolves`